sdk/textanalytics/ai-text-analytics/src/generated/generatedClientContext.ts

```diff
@@ -1,32 +1,32 @@
 /*
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
  *
  * Code generated by Microsoft (R) AutoRest Code Generator.
- * Changes may cause incorrect behavior and will be lost if the code is
- * regenerated.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
  */
 
 import * as coreHttp from "@azure/core-http";
+import { GeneratedClientOptionalParams } from "./models";
 
 const packageName = "@azure/ai-text-analytics";
-const packageVersion = "1.0.1";
+const packageVersion = "1.1.0-preview.1";
 
 export class GeneratedClientContext extends coreHttp.ServiceClient {
   endpoint: string;
 
   /**
    * Initializes a new instance of the GeneratedClientContext class.
    * @param endpoint Supported Cognitive Services endpoints (protocol and hostname, for example:
-   * https://westus.api.cognitive.microsoft.com).
-   * @param [options] The parameter options
+   *                 https://westus.api.cognitive.microsoft.com).
```

Comment:
@joheredi there seems to be a weird space in the generated code

Reply:
Thanks @iscai-msft, if I recall correctly the general consensus when discussed this with the TS team was to align the extra lines with the start of the previous one (after the param name). @sarangan12 please consider double checking this while working on Azure/autorest.typescript/issues/550